integration-test/playwright/windows-permissions.spec.js

```diff
@@ -31,11 +31,42 @@ export class WindowsPermissionsSpec {
     }
 
     async enabled () {
+        await this.installPolyfills()
         const config = JSON.parse(readFileSync(this.config, 'utf8'))
         await this.setup({ config })
         await this.page.goto(this.htmlPage)
     }
 
+    /**
+     * In CI, the global objects such as USB might not be installed on the
+     * version of chromium running there.
+     */
+    async installPolyfills () {
+        await this.page.addInitScript(() => {
+            // @ts-expect-error - testing
+            if (typeof Bluetooth === 'undefined') {
```

Comment:
Why is this in the patch?

Reply:
I think we need to revert the Harmful API merge - I'll open it as a separate PR after I've checked with @muodov